wqflask/wqflask/markdown_routes.py

```diff
@@ -9,21 +9,22 @@
 from flask import render_template
 
 glossary_blueprint = Blueprint('glossary_blueprint', __name__)
-
+references_blueprint = Blueprint("references_blueprint", __name__)
+environments_blueprint = Blueprint("environments_blueprint", __name__)
+links_blueprint = Blueprint("links_blueprint", __name__)
+policies_blueprint = Blueprint("policies_blueprint", __name__)
+facilities_blueprint = Blueprint("facilities_blueprint", __name__)
 
 def render_markdown(file_name):
-    """Try to fetch the file name from Github and if that fails, try to
-look for it inside the file system
-
-    """
     github_url = ("https://raw.githubusercontent.com/"
                   "genenetwork/gn-docs/master/")
+
     md_content = requests.get(f"{github_url}{file_name}")
     if md_content.status_code == 200:
-        return markdown.Markdown().convert(md_content.content.decode("utf-8"))
 
+        return markdown.markdown(md_content.content.decode("utf-8"), extensions=['tables'])
+        # return markdown.Markdown().convert(md_content.content.decode("utf-8"))
```

Comment:
**issue:** Have you tested this out using your guix profile?

I'd have to look at the docs more keenly. Also, please remove the commented out code. We don't really need that.

Reply:
Yes, it works.Maybe this could help [table-extension](https://python-markdown.github.io/extensions/tables/)